windows_containers/creating_windows_machinesets/creating-windows-machineset-azure.adoc

```diff
@@ -0,0 +1,25 @@
+[id="creating-windows-machineset-azure"]
+= Creating a Windows MachineSet on Azure
+include::modules/common-attributes.adoc[]
+:context: creating-windows-machineset-azure
+
+toc::[]
+
+:FeatureName: Windows containers
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+You can create a Windows MachineSet to serve a specific purpose in your {product-title} cluster on Microsoft Azure. For example, you might create infrastructure Windows MachineSets and related Machines so that you can move supporting Windows workloads to the new Windows Machines.
+
+[discrete]
+== Prerequisites
+
+* You installed the Windows Machine Config Operator (WMCO) using Operator Lifecycle Manager (OLM).
+* You are using a Windows Server 2019 container as the OS image with the Docker container runtime add-on enabled.
```

Comment:
`Windows Server 2019` will suffice.

Reply:
@aravindhp On Slack, you mentioned:

>We are going to support `Windows Server LTSC: Windows Server 2019` and
`Windows Server SAC:  Windows Server ver 1909`.

Should those be specifically documented as the only supported Windows OS images for OCP?

Reply:
I will let @Anandnatraj make that call. But yes, those are the versions we are testing with and I would prefer we call that out.

Reply:
@codyhoag @aravindhp pls call it out specifically. In fact, we need to call out supported platforms as well. I'll tag you on a blog post with the details